spec/features/ingredients/new_ingredient_page_spec.rb

```diff
@@ -0,0 +1,19 @@
+feature 'user visits new ingredient page' do
+  before do
+    visit('/ingredients/new')
+  end
+
+  it 'creates new ingredient' do
+    fill_in('ingredient_name', with: 'Example Ingredient')
+    fill_in('ingredient_unit_type', with: 'Type')
+    click_button('Save')
+    current_path.match(ingredients_path)
```

Comment:
We should somehow verify that our new ingredient was really created. Since we are redirected back to ingredients index page we have the name visible there.